### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="spec"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1181]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxgrid__item__bg__hx--open_standard.mxgrid__item__bg__hx > a[href$="spec"]
#### Snippet
`<a href="/docs/spec">Open specification</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.96 (foreground color: #0098d4, background color: #f4f4f4, font size: 13.5pt (18px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
